operator/internal/manifests/object_storage_test.go

```diff
@@ -188,7 +191,7 @@ func TestConfigureDeploymentForStorageType(t *testing.T) {
 		tc := tc
 		t.Run(tc.desc, func(t *testing.T) {
 			t.Parallel()
-			err := configureDeploymentForStorageType(tc.dpl, tc.storageType, tc.secretName)
+			err := storage.ConfigureDeployment(tc.dpl, tc.opts)
```

Comment:
The tests in this file now only test functions in the `storage` package. I think this file should be moved to that package as well (probably as `configure_test.go`).